modules/chat/src/main/ChatApi.scala

```diff
@@ -147,7 +147,10 @@ final class ChatApi(
         busChan: BusChan.Select
     ): Funit =
       coll.byId[UserChat](chatId.value) zip userRepo.byId(modId) zip userRepo.byId(userId) flatMap {
-        case ((Some(chat), Some(mod)), Some(user)) if isMod(mod) || scope == ChatTimeout.Scope.Local =>
+        case ((Some(chat), Some(mod)), Some(user))
+            if isMod(mod) || (busChan(BusChan) == BusChan.Study && isRelayMod(
+              mod
+            )) || scope == ChatTimeout.Scope.Local =>
```

Comment:
This technically allows timeouts in all studies but I think that's acceptable and avoiding this would require a bit of a mess as far as I am able to tell.